### ISSUE TITLE
Auto paste chuncks?

### ISSUE THREAD

**Issue by bigplayer-ai** [jupediaz/chatgpt-prompt-splitter#2]
Hey, can you make a script that Auto paste the chuncks to chatGPT??

**Comment by jupediaz** [comment]
Hello. That would be great... but I need to figure out how to do it if they are on different tabs... I would need to do some research. But I'm already preparing a version of the chatgpt-prompt-splitter as a Chrome extension, where it will be more accessible to copy+paste automatically... let's see... I'll publish if I make it work. Thanks for your interest!

**Comment by willsheppard** [comment]
Any progress on this?

**Comment by rsinohara** [comment]
No, sorry, time has been short on my side. At this point I would also do a good search, see if no one else implemented it already.
